packages/repo-tools/src/commands/api-reports/sql-reports/getPgSchemaInfo.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { Knex } from 'knex';
+import { SchemaIndexInfo, SchemaInfo, SchemaSequenceInfo } from './types';
+
+export async function getPgSchemaInfo(knex: Knex): Promise<SchemaInfo> {
+  const { rows: tableNames } = await knex.raw<{ rows: { name: string }[] }>(`
+    SELECT table_name as name
+    FROM information_schema.tables
+    WHERE
+      table_schema = 'public'
+      AND table_type = 'BASE TABLE'
+      AND table_name NOT LIKE 'knex_migrations%'
```

Comment:
Could maybe even make it `%knex%` (same below)

Reply:
Eh, I like to keep it somewhat explicit